lib/widget/club_detail/activity/activity_list_item.dart

```diff
@@ -1,75 +1,102 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_template/model/webservice/activity/activity.dart';
 import 'package:flutter_template/styles/theme_dimens.dart';
+import 'package:flutter_template/util/locale/localization.dart';
 import 'package:flutter_template/widget/general/stacked_data.dart';
 import 'package:flutter_template/widget/provider/data_provider_widget.dart';
+import 'package:flutter_template/styles/theme_durations.dart';
 
 class ActivityListItem extends StatelessWidget {
   final Activity activity;
+  final int selectedActivity;
+  final int index;
+  final Localization localization;
```

Comment:
This isn't something you want to do I guess.. 
You can get your Localization.of(context) if needed..